website/templates/project/register.mako

```diff
@@ -3,40 +3,51 @@
 
 <legend class="text-center">Register</legend>
 
-% if schema:
-    <%include file="metadata/register_${ str(metadata_version) }.mako" />
+% if node.get('registered_schema'):
+  <div id="registrationMetaDataScope" class="container scripted">
+    <div class="row">
+      <div class="span8 col-md-2 columns eight large-8">        
```

Comment:
I'm not 100% sure, but some of these CSS class names look a bit off (columns, eight, large-8). They don't seemed to be defined in bootstrap, registrations.css, or mentioned in osf-style guide.